### DISCUSSION TITLE
Parameters of scenicplus

### DISCUSSION THREAD

**Discussion by XiaoKaixuan12333** [aertslab/scenicplus#382] [Q&A]
Hi, actually, I have success run the whole tutorial of scenicplus, the output is about 25 activator eRegulons. And I really appreciate your work. 

but i was wonder whether if i can bulid  eRegulons  of some specified tfs? and if i want to output more eRegulons, which parameters should i need to change? 

please help me.
Thanks!

**Comment by SeppeDeWinter** [comment]
Hi

> but i was wonder whether if i can bulid eRegulons of some specified tfs?

This is not really possible unfortunatly.


> and if i want to output more eRegulons, which parameters should i need to change?

The motif enrichment step is most limiting. So changing the regions you use for motif enrichment might produce extra eRegulons.

Best,

Seppe

**Reply by XiaoKaixuan12333** [reply]
thanks for your reply. 

I noticed in /grn_builder/modules.py 258 line may have an error : _nn_cpu_. maybe it is _n_cpu_

**Reply by SeppeDeWinter** [reply]
Hi @XiaoKaixuan12333 

Thanks! 
It's fixed now https://github.com/aertslab/scenicplus/commit/fa55dae55203951c0c42e359b65a28010cd544f6

Best,

Seppe
